channels/ir.m3u

```diff
@@ -33,25 +33,39 @@ https://bozztv.com/36bay2/gin/giniko_tvbox-cine-series_800kb_36bay2/tracks-v1a1/
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_drama/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-junior.png?1212" group-title="Entertainment",GEM Junior
+#EXTINF:-1 tvg-id="" tvg-name="GEM Film" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_film.png?13" group-title="Entertainment",Gem Film
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_film/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Fit" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_fit.png?13" group-title="Entertainment",Gem Fit
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_fit/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Food" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_food.png?13" group-title="Entertainment",Gem Food
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_food/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemjunior.png?13" group-title="Entertainment",GEM Junior
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_junior/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Kids" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-kids.png?1212" group-title="Kids",GEM Kids
+#EXTINF:-1 tvg-id="" tvg-name="GEM Kids" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemkids.png?13" group-title="Kids",GEM Kids
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_kids/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Series" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-series.png?1212" group-title="Entertainment",Gem Series
-http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM TV" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-tv.png?1212" group-title="Entertainment",GEM TV
+#EXTINF:-1 tvg-id="" tvg-name="GEM Life" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_life.png?13" group-title="Entertainment",Gem Life
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_life/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Nature" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_nature.png?13" group-title="Entertainment",Gem Nature
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_nature/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Property" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_property.png?13" group-title="Entertainment",Gem Property
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_property/playlist_960x540_2000k.m3u8
```

Comment:
Links do not work.

Reply:
I just checked with https://castr.io/hlsplayer and they are working